azure-pipelines.yml

```diff
@@ -62,18 +61,18 @@ jobs:
 - template: chefspec-cookstyle.yml@templates
 - template: test-kitchen.yml@templates
   parameters:
-    jobName: "haswell"
+    jobName: "haswell_agent"
     kitchenFile: kitchen.yml
     venturaPlus: true
     platforms:
       - ventura-x86
-    # - ventura-arm
+      - sonoma-x86
     suites: ${{ parameters.kitchenSuites }}
 - template: test-kitchen.yml@templates
   parameters:
-    jobName: "vintage"
-    platforms:
-    - big-sur-x86
-    - monterey-x86
-    suites: ${{ parameters.kitchenSuites }}
+    jobName: "vintage_agent"
     kitchenFile: kitchen.yml
+    platforms:
+      - big-sur-x86
```

Comment:
BigSur is no longer officially supported -- does testing still need to be done on that platform?